app/migrations/Version20171227110138.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20171227110138 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE citizen_projects ADD image_name LONGTEXT DEFAULT NULL');
```

Comment:
Is it really needed to have a `LONGTEXT` field for this one ? `VARCHAR` should be enough.